charts/warpstream-agent/values.yaml

```diff
@@ -21,6 +25,13 @@ deploymentStrategy:
     # The number of pods that can be unavailable during the update process (both integers and percentages are supported)
     maxUnavailable: 1
 
+statefulSetConfig:
+  clusterDomain: cluster.local
+  podManagementPolicy: OrderedReady  # or Parallel, see https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
+  updateStrategy:
+    # see https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+    type: RollingUpdate
```

Comment:
Will this inherit the deployment strategy around maxSurge/maxUnavailable? If so can you comment that

Reply:
No maxSurge isn't a thing in statefulsets and maxUnavailable is alpha still so most cloud K8s implementations don't have it enabled.